dist/s_string.ok

```diff
@@ -945,6 +946,7 @@ mkdir
 mmap
 mmrand
 mnt
+mongodb
```

Comment:
I agree with Sulabh's comment further down - you should use MongoDB - which already has an entry in this file.